de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/joiner/JoinerNodeModel.java

```diff
@@ -596,8 +599,8 @@ private CombinedFskPortObject createCombinedFskPortObject(FskPortObject jFirstIn
   }
 
   private static FskPortObject createEmptyFSKObject() throws IOException {
-
-    FskPortObject fskPortObject = new FskPortObject(Optional.empty(), "", Collections.emptyList());
+    
+    FskPortObject fskPortObject = new FskPortObject(Optional.of(new FilesEnvironmentManager(new String[0])), "", Collections.emptyList());
```

Comment:
This challenges the approach of the environment managers. An empty manager object is more transparent as it explicitly describes a lack of environment. With an empty files environment or environment with 0 files we are hiding the fact that this environment is empty and enabling some other client code to use a manager, probably because it has not been updated to support the lack of environment.
I would rather help to update the client code to support the case of models with no environment rather than including this workaround.